api/tests/test-helper.js

```diff
@@ -38,13 +35,12 @@ function generateIdTokenForExternalUser(externalUser) {
 }
 
 async function insertUserWithRolePixMaster() {
-
   const user = databaseBuilder.factory.buildUser.withPixRolePixMaster({
     id: 1234,
     firstName: 'Super',
     lastName: 'Papa',
     email: 'super.papa@example.net',
-    password: 'abcd1234',
+    rawPassword: 'abcd1234',
```

Comment:
Chipotage
J'ai pas trouvé mieux que unhashed dans [cette RFC](https://tools.ietf.org/id/draft-farrell-decade-ni-03.html)
```suggestion
    unhashedPassword: 'abcd1234',
```

Reply:
@octo-topi je ne trouve pas le mot "unhashed" dans [la RFC en question] ?

Le mot `unhashed` à mon sens fait plutôt référence au résultat d'une attaque sur un hash, où on réussit à retrouver un mot de passe à partir d'un hash.

Le problème ici je trouve que ce n'est pas d'appeler `password` le mot de passe — après tout c'est parfaitement correct — mais plutôt d'appeler `password` le résultat de la fonction de hachage alors que tout le but est que ce ne soit précisément _pas_ un mot de passe (au sens de quelque chose qui doit absolument rester secret). Si on utilisait par exemple `passwordHash` là où on utilise `password` on n'aurait pas besoin d'ajouter `raw-`, `unhashed-`, etc.

Reply:
J'ai corrigé mon commentaire qui prêtait à confusion: même la RFC ne définit pas de terrme.
Je dis :100: à `hashedPassword` ou `hashFromPassword` plutôt que de renommer la source `plainPassword`

Reply:
Si on pinaille, je continue de préférer `passwordHash` : c'est _un hash_ du _mot de passe_, deux objets de nature très différente. `hashedPassword` m'évoque un objet qu'on aurait modifié sans changer sa nature. Par exemple si j'ai la liste `[1,2,3]` alors je trouve ça plus parlant de dire que `6` est la _somme de la liste_ plutôt que _la liste sommée_ — alors que `[1,2]` serait _la liste tronquée_…

Reply:
Done

Reply:
Implémenté dans https://github.com/1024pix/pix/pull/2310